src/ugeneui/src/Main.cpp

```diff
@@ -420,6 +420,11 @@ int main(int argc, char **argv) {
     fixMacFonts();
 #endif
 
+ #ifdef Q_OS_MACOS
```

Comment:
Replace with `Q_OS_MACX`

Reply:
No, Q_OS_MACX is already removed from Qt, Q_OS_MACOS is a correct macro.